src/reducers/application.js

```diff
@@ -126,18 +126,22 @@ export default (state = initialState, action) => {
       };
     }
     case CONTENT_TYPES_LOADED: {
+      const { data } = action.payload.contentTypes;
       return {
         ...state,
-        contentTypes: action.payload.contentTypes.data.reduce(
-          (accumulator, contentType) => ({
-            ...accumulator,
-            [contentType.attributes.type]: {
-              name: contentType.attributes.name,
-              description: contentType.attributes.description,
-            },
-          }),
-          {},
-        ),
+        contentTypes:
```

Comment:
Can we add the comment how this data could be empty? Is this a backend returning nothing?

Reply:
After the changes made for `loadContentTypes`, wouldn't we be able to revert all changes made to this file?

Reply:
Hi @lauriii  I think it is better to have validations in place to counter any unaccounted error.

Reply:
@vishalchandna1 Looks like these changes aren't actually needed anymore. The response `data` will never not be an array so having this here will just confuse people in the future.